### PR TITLE
OSIDB-2308: Fix CVSS composable

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR:
* makes variables in CVSS composable more explicit (eg. versions, issuers)
* fixes cvss data search - version and issuer need to be both specified as with only issuer specified it might happen that CVSSv2 score was found first and used

Consideration:
CVSS composable should probably be redesigned a little bit a generalized in the future as there will be CVSSv4 as well

This partially solves OSIDB-2308 (sub-task OSIDB-2389)

### CODE REVIEW

**Review by superbuggy — APPROVED** [review]
I appreciate the variable naming in particular, this rocks

**Review by tboyetteredhat — APPROVED** [review]
Nice job 🚀

File: src/composables/useCvssScoresModel.ts [hunk 1]
```diff
@@ -6,11 +6,19 @@ import type { ZodFlawType } from '@/types/zodFlaw';
...
+      (assessment) => assessment.issuer === issuer && assessment.cvss_version === version
+    );
+  }
+
   const wasCvssModified = ref(false);
-  const redHatCvssData = flaw.value.cvss_scores.find((assessment) => assessment.issuer === 'RH');
-  const flawRhCvss = ref(
-    redHatCvssData || {
+  const flawRhCvss3 = ref(
+    getCvssData('RH', 'V3') || {
```
Comment (superbuggy) [inline src/composables/useCvssScoresModel.ts:21]: Much better

**Review by superbuggy — COMMENTED** [review]
🎉 for naming specificity

**Pull request merged**